multipathd/uxclnt.c

```diff
@@ -70,20 +69,20 @@ static int need_quit(char *str, size_t len)
  */
 static void process(int fd, unsigned int timeout)
 {
-	char *line;
+	char line[256];
 	char *reply;
+	unsigned int i;
 	int ret;
+	char prompt[] = "multipathd> ";
 
 	cli_init();
-	rl_readline_name = "multipathd";
-	rl_completion_entry_function = key_generator;
-	while ((line = readline("multipathd> "))) {
+	for (i = 0; i < strlen(prompt); i++)
+		fputc(prompt[i], stdout);
```

Comment:
why not just use `fputs()`?

Reply:
Because fputs() always attaches a newline, and we want to display a prompt (ie without a newline).

Reply:
> Because fputs() always attaches a newline

`puts()` does this but `fputs()` does not.